docs/pages/blog/material-ui-2024-updates.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Material UI 2024 end-of-year updates
+description: A look through Material UI during 2024, and a sneak peek of what's to come.
+date: 2024-12-11T00:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+As the end of the year approaches, we want to take some time to review some of this year's highlights for Material UI and update you on what the community can expect moving forward.
+
+## Material UI v6 release and support
+
+We released Material UI v6 in August, offering new features, fixes, and a smoother migration experience. So far, almost a million users have successfully migrated.
+
+The reception for the new version has been very positive, and we hope many more users can migrate in the upcoming months.
+
+Our focus has been on responding to v6-related issues to support your migration. We thank everyone in the community who has helped by reporting or fixing problems and encourage everyone else to migrate. We are happy and ready to help!
+
+## React 19 support
+
+As of today, Material UI v5 and v6 support React 19. This has been one of our main focuses during the second half of 2024.
+
+Next up, we'll update Material UI's dependencies and type versions once React 19 is stable.
```

Comment:
A new v5 release will be out today so the same applies to v5.

Reply:
React 19 v5 PR is merged 🎉 Depending on the timeline, it might release before or after this blog, but it won't affect users that much. We can remove this line.